index.html

```diff
@@ -0,0 +1,89 @@
+<h3>Data collection on this website</h3>
+<h4>Who is responsible for the data collection on this website?</h4>
+<p>The data processing on this website is carried out by the website operator. You can find the contact details of the website operator in the section "Information about the responsible party" in this data protection declaration.</p>
+<h4>How do we collect your data?</h4>
+<p>On the one hand, your data is collected when you provide it to us. This can be, for example, data that you enter in a contact form.</p>
+<p>Other data is collected automatically or after your consent when you visit the website by our IT systems. This is mainly technical data (e.g. internet browser, operating system or time of page view). The collection of this data takes place automatically as soon as you enter this website.</p>
+<h4>What do we use your data for?</h4>
+<p>Some of the data is collected to ensure error-free provision of the website. Other data may be used to analyze your user behavior.</p>
+<h4>What rights do you have regarding your data?</h4>
+<p>You have the right at any time to receive information free of charge about the origin, recipient and purpose of your stored personal data. You also have a right to request the correction or deletion of this data. If you have given your consent to data processing, you can revoke this consent at any time for the future. You also have the right to request the restriction of the processing of your personal data under certain circumstances. Furthermore, you have the right to lodge a complaint with the competent supervisory authority.</p>
+<p>You can contact us at any time with regard to this and other questions on the subject of data protection.</p>
+
+<h2>2. General notes and mandatory information</h2>
+<h3>Data privacy</h3>
+<p>The operators of these pages take the protection of your personal data very seriously. We treat your personal data confidentially and in accordance with the statutory data protection regulations and this privacy policy.</p>
+<p>When you use this website, various personal data are collected. Personal data is data with which you can be personally identified. This privacy policy explains what data we collect and what we use it for. It also explains how and for what purpose this is done.</p>
+<p>We point out that data transmission over the Internet (eg communication by e-mail) security gaps. A complete protection of the data against access by third parties is not possible.</p>
+<h3>Note on the responsible body</h3>
+<p>The responsible party for data processing on this website is:</p>
+<p>Institute for Software Engineering, Dept. of Software Quality & Architecture at the University of Stuttgart
+    Universitätsstraße 38, 70569 Stuttgart</p>
+<p>Phone: <a href="tel:+4971168588273">+49 711 685 88273</a> <br>
+    E-mail: <a href="mailto:steffen.becker@iste.uni-stuttgart.de">steffen.becker@iste.uni-stuttgart.de</a></p>
```

Comment:
1. Bad idea to put personal emails statically into the text. That opens up the recipient to spam bot attacks.
2. The responsible party is not always Professor Becker. It is only Professor Becker when he hosts it. It's always the one hosting it.
3. This means we need a mechanism for inserting variables into the HTML file. PHP?
4. I recommend a force-push to get rid of these explicit addresses.

Reply:
We are unfortunately obliged to include concrete data, so we can't get rid of them completely + the data is 1 to 1 as static as here on the uni website which is why we don't need a force push in my opinion.